cranelift-wasm/src/environ/spec.rs

```diff
@@ -600,6 +601,29 @@ pub trait ModuleEnvironment<'data>: TargetEnvironment {
         elements: Box<[FuncIndex]>,
     ) -> WasmResult<()>;
 
+    /// Declare a passive element segment.
+    fn declare_passive_element(
+        &mut self,
+        index: PassiveElemIndex,
+        elements: Box<[FuncIndex]>,
+    ) -> WasmResult<()>;
+
+    /// Provides the number of passive data segments up front.
+    ///
+    /// By default this does nothing, but implementations may use this to
+    /// pre-allocate memory if desired.
+    fn reserve_passive_data(&mut self, count: u32) -> WasmResult<()> {
+        let _ = count;
```

Comment:
What's idiomatic rust for unused function arguments? In `dummy.rs` I see unused function arguments prefixed with `_`, and here we're adding a `_` use inside of the function def. I'm fine with either, but would prefer it to be consistent.

Reply:
Prefixing the argument with `_` is the most common. (I haven't every seen it this way :) )

Reply:
It is true that one would usually use `_prefix`, but because it is in a trait method's default implementation, that underscore would become part of the variable names in the auto-generated rustdocs. The separate underscore is to avoid underscore-prefixed variable names in the rustdocs.